sigs.yml

```diff
@@ -643,7 +643,7 @@
     - name: Daniel Gomez Blanco
       github: danielgblanco
   - name: Security
-    meeting: Bi-weekly on Wednesday at 14:00 UTC
+    meeting: Every other Wednesday at 07:00 PT
```

Comment:
@open-telemetry/sig-security-maintainers please take a look if I did the math for the timezone correctly

Reply:
Yup this is correct